apps/src/templates/rubrics/RubricSettings.jsx

```diff
@@ -178,9 +184,29 @@ export default function RubricSettings({
         [style.settingsHidden]: !visible,
       })}
     >
-      <Heading2>{i18n.settings()}</Heading2>
+      {!experiments.isEnabled('ai-rubrics-redesign') && (
+        <Heading2>{i18n.settings()}</Heading2>
+      )}
+      {experiments.isEnabled('ai-rubrics-redesign') && (
+        <div className={style.studentInfoGroup}>
+          <Heading5>
+            {i18n.lessonNumbered({
+              lessonNumber: lesson.position,
+              lessonName: lesson.name,
+            })}
+          </Heading5>
+          <div className={style.selectors}>
+            <SectionSelector reloadOnChange={true} requireSelection={false} />
+          </div>
+        </div>
+      )}
       {canProvideFeedback && (
         <div className={style.aiAssessmentOptions}>
+          {/* {experiments.isEnabled('ai-rubrics-redesign') && (
+            <div className={style.selectors}>
+              <SectionSelector reloadOnChange={true} requireSelection={false} />
+            </div>
+          )} */}
```

Comment:
I assume this is here for continuing work? If not, it's probably best to remove it before we merge.